src/jit/utils.cpp

```diff
@@ -1614,3 +1614,32 @@ unsigned __int64 FloatingPointUtils::convertDoubleToUInt64(double d) {
 
     return u64;
 }
+
+// Rounds a double-precision floating-point value to the nearest integer,
+// and rounds midpoint values to the nearest even number.
+// Note this should align with classlib in floatnative.cpp
+// Specializing for x86 using a x87 instruction is optional since
+// this outcome is identical across targets.
+double FloatingPointUtils::round(double d)
+{
+    double tempVal;
+    double flrTempVal;
```

Comment:
IMO it's preferable to move these variable declarations where they are initialized.